galaxy_ng/app/access_control/statements/insights.py

```diff
@@ -234,6 +234,12 @@
             "effect": "allow",
             "condition": "has_model_perms:galaxy.view_group"
         },
+        {
+            "action": "create",
```

Comment:
Another change: add `delete` here as well.

The way that this works in pulp is that there's a Role table and a GroupRole table. The GroupRole table tracks associations between Roles and Groups. What this access policy here is actually doing is allowing users to create, view and remove role/group associations, not the roles themselves.